reactjs/src/scenes/Roles/index.tsx

```diff
@@ -101,7 +109,12 @@ class Role extends AppComponentBase<IRoleProps> {
 
   saveFormRef = (formRef: any) => {
     this.formRef = formRef;
-  };
+    };
+
+    handleChange(event: any) {
+        this.state.filter = event.target.value;
```

Comment:
You can change it arrow function following:

handleChange = (event: any) => {
    this.setState({ filter: event.target.value });
}

If Search component used then 
handleSearch = (event: any) => {
    this.setState({ filter: event.target.value });
}

Reply:
If Search component used then can be modified as follows:
handleSearch = (value: string) => {
this.setState({ filter: value }, async () => await this.getAll());
}